plugins/slicer/MONAILabel/MONAILabel.py

```diff
@@ -1796,6 +1796,7 @@ def onUpdateScribbles(self):
 
             # save scribbles + label to file
             slicer.util.saveNode(labelmapVolumeNode, scribbles_in)
+            slicer.mrmlScene.RemoveNode(labelmapVolumeNode)
```

Comment:
It is just a temporary labelmap volume node that is created 20 lines above (`labelmapVolumeNode = slicer.mrmlScene.AddNewNodeByClass("vtkMRMLLabelMapVolumeNode")).

The segmentation could be exported to file without creating a temporary node (for example using [ExportSegmentsBinaryLabelmapRepresentationToFiles])https://github.com/Slicer/Slicer/blob/d88bfaec0e884eb532c2d3c623a1eb59ba02cccd/Modules/Loadable/Segmentations/Logic/vtkSlicerSegmentationsModuleLogic.h#L272-L286), or lower level APIs that allow more customization. But creating a temporary node and deleting it is not a very heavy operation, so that should be fine, too.

What I would find significantly better is to switch to a file format that contain description of the segments (which label value correspond to what) and supports overlapping segments. These requirements are fulfilled by the [.seg.nrrd](https://slicer.readthedocs.io/en/latest/developer_guide/modules/segmentations.html#segmentation-labelmap-file-format-seg-nrrd) (already well established and proven to work robustly in Slicer, but not a wide community standard) or [OME-Zarr](https://ngff.openmicroscopy.org/latest) (being worked on by a large community, but still evolving) file formats.

Reply:
I can confirm that the .seg.nrrd file format has some advantages. We have used it in MONAILabel/Slicer for three different segmentation tasks and have now segmented several thousand X-ray images from scratch. Model training with MONAI core was no problem. We successfully trained several Unets with support for overlapping segments. 

We then used these models successfully in MONAILabel for pre-segmentations. To get the generated .seg.nrrd files automatically back into Slicer after inference, we had to adjust the MONAILabel extension in updateSegmentationMask().

I haven't found the time yet but actually planned to create a separate issue for this.

Reply:
> why you want to remove the labelmap node? this will also remove all the other labels which user might be working on..? is it not true?

It is a temporary volume as @lassoan correctly pointed out. Thanks for the merge.